app/src/main/java/net/gsantner/markor/frontend/textview/HighlightingEditor.java

```diff
@@ -190,6 +191,11 @@ private int[] hlRegion(final Rect rect) {
         }
     }
 
+    @Override
+    public boolean bringPointIntoView(int i) {
+        return super.bringPointIntoView(i);
+    }
```

Comment:
Is there a reason for this? As far I see it's a override that just calls the original implementation